src/database/entities/microservice/january.rs

```diff
@@ -88,11 +88,30 @@ pub struct Metadata {
     color: Option<String>,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct BotEmbed {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    icon_url: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    url: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    title: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    description: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    image: Option<Image>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    video: Option<Video>,
```

Comment:
I'm not sure whether we can trust clients to accurately deliver this information

Reply:
Should we just make people pass a url to an image/video instead?

Reply:
currently blocked by https://github.com/revoltchat/january/issues/4.